src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const headers = [...table.querySelectorAll('th')];
+const tbody = table.querySelector('tbody');
+
+const rows = [...tbody.querySelectorAll('tr')];
+
+const getNumber = (n) => Number(n.replace(/[^0-9.-]+/g, ''));
+
+headers.forEach((header, i) => {
+  header.addEventListener('click', () => {
+    rows.sort((a, b) => {
+      const cellA = a.querySelectorAll('td')[i].textContent;
+      const cellB = b.querySelectorAll('td')[i].textContent;
+
+      if (Number(getNumber(cellA))) {
+        return getNumber(cellA) - getNumber(cellB);
```

Comment:
The condition `if (Number(getNumber(cellA)))` is used to check if the cell contains a number. However, this condition might not work as expected for cells with a value of '0' or empty strings, as `Number('0')` is falsy. Consider using a more explicit check, such as `!isNaN(getNumber(cellA))`, to determine if the cell contains a numeric value.